Error during CMOR installation

I have  errors when I try to instal CMOR using bash (it also happened under tcsh). ZDuring installation in bash the command  
**python run_tests.py -v2 -H -n1 Test/test_python_CMIP6_CV*.py**  gives error summary
Ran 38 tests, 14 failed (63.16% success)
Failed tests:
         Test/test_python_CMIP6_CV_trackingNoprefix.py
         Test/test_python_CMIP6_CV_badsource.py
         Test/test_python_CMIP6_CV_forcenoparent.py
         Test/test_python_CMIP6_CV_sub_experimentnotset.py
         Test/test_python_CMIP6_CV_HISTORY.py
         Test/test_python_CMIP6_CV_fxtable.py
         Test/test_python_CMIP6_CV_hierarchicalattr.py
         Test/test_python_CMIP6_CV_trackingprefix.py
         Test/test_python_CMIP6_CV_externalvariables.py
         Test/test_python_CMIP6_CV_unicode.py
         Test/test_python_CMIP6_CV_furtherinfourl.py
         Test/test_python_CMIP6_CV_parentmipera.py
         Test/test_python_CMIP6_CV_sub_experimentbad.py
         Test/test_python_CMIP6_CV_badinstitution.py


Further when I run **make test -o cmor -o python** it gives the next error:
...
writing time: 3 0.75
wrote var 1 time: 3
File: CMIP6/CMIP6/ISMIP6/PCMDI/PCMDI-test-1-0/piControl-withism/r3i1p1f1/Omon/sos/gn/v20210921/sos_Omon_PCMDI-test-1-0_piControl-withism_r3i1p1f1_gn_201001-201001.nc
File: CMIP6/CMIP6/ISMIP6/PCMDI/PCMDI-test-1-0/piControl-withism/r3i1p1f1/Omon/sos/gn/v20210921/sos_Omon_PCMDI-test-1-0_piControl-withism_r3i1p1f1_gn_201004-201004.nc
File: CMIP6/CMIP6/ISMIP6/PCMDI/PCMDI-test-1-0/piControl-withism/r3i1p1f1/Omon/sos/gn/v20210921/sos_Omon_PCMDI-test-1-0_piControl-withism_r3i1p1f1_gn_201007-201007.nc
E
======================================================================
ERROR: testCMIP6 (__main__.TestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/san/CMIP6_work/cmor/Test/test_python_sos_psu_units.py", line 103, in tearDown
    shutil.rmtree("./CMIP6")
  File "/home/san/anaconda/envs/CMOR/lib/python3.9/shutil.py", line 726, in rmtree
    _rmtree_safe_fd(fd, path, onerror)
  File "/home/san/anaconda/envs/CMOR/lib/python3.9/shutil.py", line 663, in _rmtree_safe_fd
    _rmtree_safe_fd(dirfd, fullname, onerror)
  File "/home/san/anaconda/envs/CMOR/lib/python3.9/shutil.py", line 663, in _rmtree_safe_fd
    _rmtree_safe_fd(dirfd, fullname, onerror)
  File "/home/san/anaconda/envs/CMOR/lib/python3.9/shutil.py", line 663, in _rmtree_safe_fd
    _rmtree_safe_fd(dirfd, fullname, onerror)
  [Previous line repeated 7 more times]
  File "/home/san/anaconda/envs/CMOR/lib/python3.9/shutil.py", line 683, in _rmtree_safe_fd
    onerror(os.unlink, fullname, sys.exc_info())
  File "/home/san/anaconda/envs/CMOR/lib/python3.9/shutil.py", line 681, in _rmtree_safe_fd
    os.unlink(entry.name, dir_fd=topfd)
OSError: [Errno 16] Device or resource busy: '.nfs000000002c0841b80001cdd3'
----------------------------------------------------------------------
Ran 1 test in 0.704s
FAILED (errors=1)
make[1]: *** [test_a_python] Error 1
make[1]: Leaving directory `/home/san/CMIP6_work/cmor'
make: *** [test_python] Error 2

So, I can not install CMOR. Can  you explain what is going on and how can I avoid it?

-- Thanks

--- a/Test/test_python_sos_psu_units.py
+++ b/Test/test_python_sos_psu_units.py
@@ -100,7 +100,7 @@ class TestCase(unittest.TestCase):
 
     def tearDown(self):
         import shutil
-        shutil.rmtree("./CMIP6")
+        # shutil.rmtree("./CMIP6")
 
 
 if __name__ == '__main__':

